### ISSUE TITLE
simpletransformers does not work for multi class classification after recent update

### ISSUE THREAD

**Issue by CharmyWee** [ThilinaRajapakse/simpletransformers#945]
After the recent update [11/01/2021], Simple Transformer multi class classification gives "ValueError: multiclass format is not supported" error

**To Reproduce**
Run the below code which I have extracted from SimpleTransformer documentation guide>>  https://simpletransformers.ai/docs/multi-class-classification/
```
pip install simpletransformers
from simpletransformers.classification import ClassificationModel, ClassificationArgs
import pandas as pd
import logging
logging.basicConfig(level=logging.INFO)
transformers_logger = logging.getLogger("transformers")
transformers_logger.setLevel(logging.WARNING)

# Preparing train data
train_data = [
    ["Aragorn was the heir of Isildur", 1],
    ["Frodo was the heir of Isildur", 0],
    ["Pippin is stronger than Merry", 2],
]
train_df = pd.DataFrame(train_data)
train_df.columns = ["text", "labels"]

# Preparing eval data
eval_data = [
    ["Aragorn was the heir of Elendil", 1],
    ["Sam was the heir of Isildur", 0],
    ["Merrry is stronger than Pippin", 2],
]
eval_df = pd.DataFrame(eval_data)
eval_df.columns = ["text", "labels"]

# Optional model configuration
model_args = ClassificationArgs(num_train_epochs=1)
model_args = {
    "overwrite_output_dir": True,
  }

# Create a ClassificationModel
model = ClassificationModel(
    'bert',
    'bert-base-cased',
    num_labels=3,
    args=model_args,
    use_cuda =False,
) 

# Train the model
model.train_model(train_df)

# Evaluate the model
result, model_outputs, wrong_predictions = model.eval_model(eval_df)
```
And it gives the following error
```
ValueError                                Traceback (most recent call last)
<ipython-input-9-c3fe037eb272> in <module>()
     45 
     46 # Evaluate the model
---> 47 result, model_outputs, wrong_predictions = model.eval_model(eval_df)

4 frames
/usr/local/lib/python3.6/dist-packages/sklearn/metrics/_ranking.py in _binary_clf_curve(y_true, y_score, pos_label, sample_weight)
    534     if not (y_type == "binary" or
    535             (y_type == "multiclass" and pos_label is not None)):
--> 536         raise ValueError("{0} format is not supported".format(y_type))
    537 
    538     check_consistent_length(y_true, y_score, sample_weight)

ValueError: multiclass format is not supported
```
**Expected behavior**
The code was working fine before the recent update.

**Screenshot**
![SimpleTransformersMultiClassError](https://user-images.githubusercontent.com/77287347/104209229-71056900-5457-11eb-9328-f2f37d4bccfb.PNG)

**Comment by ThilinaRajapakse** [comment]
This should be fixed now in (0.51.13)

**Comment by CharmyWee** [comment]
Thank you very much. Highly appreciate your quick response.
